persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -122,23 +127,28 @@ public void execute() {
 
         try  {
             balProject = ProjectLoader.loadProject(Paths.get(sourcePath));
-            name = balProject.currentPackage().descriptor().org().value() + "." + balProject.currentPackage()
-                    .descriptor().name().value() + "." + "clients";
-
             balProject = BuildProject.load(Paths.get(sourcePath).toAbsolutePath());
             balProject.currentPackage().getCompilation();
-            configurations = SyntaxTreeGenerator.readToml(Paths.get(this.sourcePath, this.configPath), name);
+            persistConfigurations = SyntaxTreeGenerator.readToml(Paths.get(this.sourcePath, PERSIST,
+                    this.persistConfigPath));
+            for (String key : persistConfigurations.keySet()) {
+                if (Pattern.matches(PLACEHOLDER_PATTERN, persistConfigurations.get(key))) {
+                      populatePlaceHolder(persistConfigurations);
+                      break;
+                }
+            }
```

Comment:
Much better logic would be,
1. read persist.toml
2. Filter all configurations which match the template regex
3. If filteredConfigs.size() >0
           readConfig.toml
           populateTemplatedConfigs(filteredConfigs)

Reply:
updated in [0b4967f](https://github.com/ballerina-platform/persist-tools/pull/54/commits/0b4967f1e318692ce31209fe0643a8f6416b6b63)

Reply:
Updated in PR PRAM https://github.com/ballerina-platform/persist-tools/pull/65